File: app/src/main/java/de/tum/in/tumcampusapp/models/tumo/IdentitySet.java
package de.tum.in.tumcampusapp.models.tumo;

import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;

import java.util.List;

@Root(name = "rowset")
public class IdentitySet {

    @ElementList(inline = true)
    private List<Identity> ids;

    @Override
    public String toString() {
        if (ids == null || ids.isEmpty()) {
            return null;
        }
        return ids.get(0)
                  .toString();
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/models/tumo/IdentitySet.kt
package de.tum.`in`.tumcampusapp.models.tumo

import org.simpleframework.xml.ElementList
import org.simpleframework.xml.Root

@Root(name = "rowset")
data class IdentitySet(@field:ElementList(inline = true)
                       var ids: List<Identity> = emptyList()) {

    override fun toString(): String {
        return if (ids.isEmpty())
            "null"
        else ids[0].toString()
    }
}
